src/test/java/com/github/wovnio/wovnjava/QueryUrlLanguagePatternHandlerTest.java

```diff
@@ -59,8 +85,20 @@ public void testGetLang__MatchingQuery__ValidSupportedLang__ReturnTargetLangObje
         assertEquals(this.french, sut.getLang("en.site.com/es/page/index.html?wovn=fr"));
     }
 
+    public void testGetLang__MatchingQuery__ValidSupportedLang__ReturnTargetLangObjectOfChinese() {
```

Comment:
These tests are hard to name, but I would call this one `testGetLang__MatchingQuery__ValidSupportedRegionalLang__ReturnTargetLangObject`
just so it isn't specifically about Chinese.

Reply:
Actually, you could just put these cases inside of the test above it as well. Similar to 
`testGetLang__MatchingSubdomain__ValidSupportedLang__ReturnTargetLangObject`